packages/pulp/pulp.spec

```diff
@@ -473,29 +473,31 @@ Pulp provides replication, access, and accounting for software repositories.
 %defattr(-,root,root,-)
 %{_usr}/lib/tmpfiles.d/
 %endif
-# 640 root:apache
-%defattr(640,root,apache,-)
+# 640 root:pulp
+%defattr(640,root,pulp,-)
 %ghost %{_sysconfdir}/pki/%{name}/ca.key
 %ghost %{_sysconfdir}/pki/%{name}/ca.crt
 %ghost %{_sysconfdir}/pki/%{name}/rsa.key
 %ghost %{_sysconfdir}/pki/%{name}/rsa_pub.key
 %config(noreplace) %{_sysconfdir}/%{name}/server.conf
-# - apache:apache
-%defattr(-,apache,apache,-)
+# - apache:pulp
+%defattr(-,apache,pulp,-)
 %dir %{_var}/lib/%{name}
 %{_var}/lib/%{name}/published
 %{_var}/lib/%{name}/static
 %{_var}/lib/%{name}/uploads
 %{_var}/www/pub
 %{_var}/cache/%{name}/
 %{_var}/run/%{name}/
-%defattr(640,apache,apache,750)
+%defattr(640,apache,pulp,750)
 %dir %{_var}/log/%{name}
 # Install the docs
 %defattr(-,root,root,-)
 %doc README LICENSE COPYRIGHT
 
 %pre server
+/usr/sbin/groupadd -f pulp
```

Comment:
https://fedoraproject.org/wiki/Packaging:UsersAndGroups#Dynamic_allocation suggests to first check if the group exists. You should also add a `Requires(pre): shadow-utils`